### PR TITLE
Searching for a sealed case by docket number bug

### PR DESCRIPTION
**Pull request by mrinsin**
Bug fix: searching for a sealed case in public case search by docket number displays 'No Matches Found'

image

### CODE REVIEW

File: shared/src/business/useCases/getCaseInteractor.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+const {
+  getDocumentContentsForDocuments,
+} = require('../useCaseHelper/case/getDocumentContentsForDocuments');
```
Comment (rachaelparris) [inline shared/src/business/useCases/getCaseInteractor.js]: Sometimes we put use case helpers on the application context and call them that way rather than importing directly (so we can mock them out). I'm not saying this is wrong or should be changed, just noting that it's different.

File: web-client/integration-tests-public/unauthedUserSearchesForCase.test.js [hunk 2]
```diff
@@ -24,12 +24,13 @@ import { unauthedUserSearchesByMeta } from './journey/unauthedUserSearchesByMeta
 import { unauthedUserViewsCaseDetail } from './journey/unauthedUserViewsCaseDetail';
 import { unauthedUserViewsPrintableDocketRecord } from './journey/unauthedUserViewsPrintableDocketRecord';
 
-const test = setupTest({
+let test = setupTest({
```
Comment (rachaelparris) [inline web-client/integration-tests-public/unauthedUserSearchesForCase.test.js]: Why were these changed to `let`?

File: web-api/serverless-public-api.yml [hunk 3]
```diff
@@ -140,6 +140,14 @@ functions:
           method: get
           cors: ui-public-${self:provider.stage}.ef-cms.${opt:domain}
 
+  getCaseForPublicDocketSearch:
```
Comment (rachaelparris) [inline web-api/serverless-public-api.yml:143]: Just a note to remember to add this endpoint to swagger :)

**Review by rachaelparris — APPROVED** [review]

File: shared/src/business/useCases/getCaseInteractor.js (hunk 1 above)
Comment (mrinsin) [reply shared/src/business/useCases/getCaseInteractor.js]: Thanks for telling me about that - I'll update the code to do that!

File: web-client/integration-tests-public/unauthedUserSearchesForCase.test.js (hunk 2 above)
Comment (mrinsin) [reply web-client/integration-tests-public/unauthedUserSearchesForCase.test.js]: This is leftover from some debugging I was doing, thanks for pointing it out!

File: web-api/serverless-public-api.yml (hunk 3 above)
Comment (mrinsin) [reply web-api/serverless-public-api.yml:143]: Thank you!
